docs(theme.md): Fix js & ts conflicts / inaccuracy (#16123)
vuetifyjs/vuetify: packages/docs/src/pages/en/features/theme.md

  - /getting-started/wireframes/
---

> <script setup>

Reviewer: As per your previous PR's, please remove this and the tabs/window code.
We do need to support TS and JS simultaneously, but it still needs to be implemented at a global scale (which is outside of the scope of just this file) For the time being, it should suffice to put the JS and TS code blocks next to each other

Author: This should be resolved sorry for my very bad memory and thank you for reminding me once again.

This makes it possible for Vuetify to implement Material Design concepts such as elevated surfaces having a lighter overlay color the higher up they are. Find out more about dark themes on the official [Material Design](https://material.io/design/color/dark-theme.html) page.

```js { resource="src/plugins/vuetify.js" }
import { createApp } from 'vue'
import { createVuetify, ThemeDefinition } from 'vuetify'

const myCustomLightTheme = {
  dark: false,
  colors: {
    background: '#FFFFFF',
    surface: '#FFFFFF',
    primary: '#6200EE',
    'primary-darken-1': '#3700B3',
    secondary: '#03DAC6',
    'secondary-darken-1': '#018786',
    error: '#B00020',
    info: '#2196F3',
    success: '#4CAF50',
    warning: '#FB8C00',
  }
}

export default createVuetify({
  theme: {
    defaultTheme: 'myCustomLightTheme',
    themes: {
      myCustomLightTheme,
    }
  }
})
```

### Typescript

Example with only the **defaultTheme** value

```ts { resource="src/plugins/vuetify.ts" }
import { createApp } from 'vue'
import { createVuetify } from 'vuetify'

export default createVuetify({
  theme: {
    defaultTheme: 'dark'
  }
})
```

> Adding new themes is as easy as defining a new property in the `theme.themes` object. A theme is a collection of colors and options that effects the overall look and feel of your application. You can use the **dark** property to designate the theme as being a **light** or **dark** variation.This makes it possible for Vuetify to implement Material Design concepts such as elevated surfaces having a lighter overlay color the higher up they are. Find out more about dark themes on the official [Material Design](https://material.io/design/color/dark-theme.html) page.

Reviewer 2: ```suggestion
```

> When using Typescript, the `ThemeDefinition` type can be used to get type hints for the structure of the theme object.

Reviewer 2: I don't agree with this change in sentence tense.

Use the `ThemeDefinition` type to get type hints for the structure of the theme object.

```js { resource="src/plugins/vuetify.js" }
import { createApp } from 'vue'
import { createVuetify, ThemeDefinition } from 'vuetify'

const myCustomLightTheme = {
  dark: false,
  colors: {
    background: '#FFFFFF',
    surface: '#FFFFFF',
    primary: '#6200EE',
    'primary-darken-1': '#3700B3',
    secondary: '#03DAC6',
    'secondary-darken-1': '#018786',
    error: '#B00020',
    info: '#2196F3',
    success: '#4CAF50',
    warning: '#FB8C00',
  }
}

export default createVuetify({
  theme: {
    defaultTheme: 'myCustomLightTheme',
    themes: {
      myCustomLightTheme,
    }
  }
})
```

  </v-window-item>
  
  <v-window-item value="ts">

Vuetify comes with two themes pre-installed, `light` and `dark`. To set the default theme of your application, use the **defaultTheme** option.

```ts { resource="src/plugins/vuetify.ts" }
import { createApp } from 'vue'
import { createVuetify } from 'vuetify'

export default createVuetify({
  theme: {
    defaultTheme: 'dark'
  }
})
```

> Adding new themes is as easy as defining a new property in the **theme.themes** object. A theme is a collection of colors and options that change the overall look and feel of your application. One of these options designates the theme as being either a **light** or **dark** variation. This makes it possible for Vuetify to implement Material Design concepts such as elevated surfaces having a lighter overlay color the higher up they are. Find out more about dark themes on the official [Material Design](https://material.io/design/color/dark-theme.html) page.

Reviewer: ```suggestion
Adding new themes is as easy as defining a new property in the `theme.themes` object. A theme is a collection of colors and options that effects the overall look and feel of your application. You can use the **dark** property to designate the theme as being a **light** or **dark** variation. This makes it possible for Vuetify to implement Material Design concepts such as elevated surfaces having a lighter overlay color the higher up they are. Find out more about dark themes on the official [Material Design](https://material.io/design/color/dark-theme.html) page.
```

> Use the `ThemeDefinition` type to get type hints for the structure of the theme object.

Reviewer: ```suggestion
When using Typescript, the `ThemeDefinition` type can be used to get type hints for the structure of the theme object.
```

Author: Hello mate, I'm terribly sorry for my bad memory haha.
I will resolve this asap! :)

## Changing theme

> This can be used to dynamically change the theme during runtime instead of having it statically chosen.

Reviewer 2: Same thing here, the tense of the sentence shouldn't change.

</script>
```

> Although it is note worthy that most components support the **theme** prop. When used a new context is created for _that_ specific component and **all** of its children. In the following example, the [v-btn](/components/buttons/) uses the **dark** theme because it is applied to its parent [v-card](/components/cards/).

Reviewer 2: This reads grammatically worse.
